src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1286,19 +1286,21 @@ private void declareModule(
     emitNoSpace("' {");
     indent();
     emitBreak();
+    // This name is offered for auto-import.
```

Comment:
Maybe expand on this a bit? "Use the proper type name as the alias name, so that TypeScript language service offers it as an auto-import"?

Reply:
Done